FirebasePerformance/Sources/Loggers/FPRGDTEvent.m

```diff
@@ -49,7 +54,30 @@ - (instancetype)initForPerfMetric:(FPRMSGPerfMetric *)perfMetric {
 #pragma mark - GDTCOREventDataObject protocol methods
 
 - (NSData *)transportBytes {
-  return [self.metric data];
+  pb_ostream_t sizestream = PB_OSTREAM_SIZING;
+
+  // Encode 1 time to determine the size.
```

Comment:
Since this is just needed once, should we make this static and measured just once?

Reply:
Could you elaborate on this? I am not so clear. `sizestream` is a local variable, do you want to make it static? `transportBytes` is a instance method, we cannot make `sizestream` static.